datapreprocess/ray/tokenize_shuffle.py

```diff
@@ -4,10 +4,12 @@
 import io
 import json
 import os
+import glob
 import resource
 import tarfile
 import time
 import traceback
+import simdjson
```

Comment:
need to be added to the requirements.txt?

Reply:
Thanks! Added.